### PR TITLE
[CARBONDATA-CI]Enabled travis CI

### PR DESCRIPTION
**Pull request by ravipesala**
Enabling travis CI for carbondata

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/53/

**Comment by chenliang613** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language: java
+script: mvn -U -Pspark-2.0 test
```
Comment (ravipesala) [inline .travis.yml]: Already with default profile our CI is running , so it is better to run this travis CI with spark2.0 profile to verify.

**Comment by ravipesala** [comment]
Updated as per comments

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**